script/lib/transifex.py

```diff
@@ -175,13 +175,17 @@ def get_grd_message_string_tags(grd_file_path):
       children = [child for child in children if child.tag == 'message']
       for child in children:
         output_elements.append(child)
+    elif element.tag == 'part': # will be handled below
+      continue
     else:
       assert False, ('Unexpected tag name %s' % element.tag)
 
   elements = lxml.etree.parse(grd_file_path).findall('.//part')
   for element in elements:
     grd_base_path = os.path.dirname(grd_file_path)
     grd_part_filename = element.get('file')
+    if grd_part_filename == 'chromeos_strings.grdp':
```

Comment:
I don't think we care about anything for chromeos, or is this needed to avoid breaking something?

Reply:
this file is explicitly ignore here to avoid this script to try to process(read) this file, which we don't copy it at all into our src and caused read error.